### ISSUE TITLE
Multiple carousels on a page using "asNavFor" option as navigation (Vanilla JS)

### ISSUE THREAD

**Issue by frazerf** [metafizzy/flickity#695]
I've got multiple carousels on a page and each carousel then has another navigation carousel which controls which slide in shown in the main carousel using the "asNavFor" option. However, when I click on the navigation carousel it changes all the slides at once instead of just the individual carousel it relates to. I've put it in it's own "container.querySelector" but this doesn't seem to be making a difference. I'm sure I'm missing something but I can't see to figure out just what.

JSfiddle here - https://jsfiddle.net/a8zucma7/

**Comment by frazerf** [comment]
Turns out I was doing a few things wrong. I've managed to come up with a solution - https://jsfiddle.net/5g4ch0g9/20/ - I don't know if it's the most elegant but it does the trick for anyone who has a similar problem going forward. 

Turns out the **asNavFor** option's defualt setting is - **document**.querySelector('.carousel') - which doesn't work if you have multiple carousels with the same class on a page. I simply changed it to - asNavFor: **container**.querySelector('.carousel'), - in the options and voila!
